engine/api/api_routes.go

```diff
@@ -148,6 +148,8 @@ func (api *API) InitRouter() {
 	r.Handle("/mon/errors/{uuid}", ScopeNone(), r.GET(api.getErrorHandler, NeedAdmin(true)))
 	r.Handle("/mon/panic/{uuid}", ScopeNone(), r.GET(api.getPanicDumpHandler, Auth(false)))
 
+	r.Handle("/help", ScopeNone(), r.GET(api.getHelpHandler, Auth(false)))
```

Comment:
```suggestion
	r.Handle("/help", ScopeNone(), r.GET(api.getHelpHandler))
```

Reply:
i would keep authentication, if the help message contains contact information, we may only want to give them to registered users